src/main/java/com/epam/spring/homework2/bean/BeanB.java

```diff
@@ -0,0 +1,42 @@
+package com.epam.spring.homework2.bean;
+
+import com.epam.spring.homework2.validation.annotation.IsInRange;
+import com.epam.spring.homework2.validation.annotation.IsNotNull;
+import com.epam.spring.homework2.validation.annotation.ValidationRequired;
+import org.springframework.beans.factory.annotation.Value;
+
+@ValidationRequired
+public class BeanB {
+
+    @IsNotNull
+    @Value("${beanB.name}")
```

Comment:
This is not a good approach. Please rework:
- make class immutable
- inject values from properties file in a method inside configuration class

Apply to all relevant classes